src/rpc/rawtransaction_util.cpp

```diff
@@ -154,6 +154,13 @@ CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniVal
         rawTx.nLockTime = nLockTime;
     }
 
+    if (!version.isNull()) {
+        int64_t nVersion = version.getInt<int64_t>();
```

Comment:
Why `int64_t`? `version` has type `uint32_t`.

Reply:
@glozow 
Thanks for the feedback. `int64_t` was my bad. However `version` field of raw transaction seems to be `int32_t` type according to [bitcoint developer doc.](https://developer.bitcoin.org/reference/transactions.html#raw-transaction-format). The doc explicitly states that the field is signed. Should I used unsigned instead?

Reply:
These docs are indeed outdated, last update was 4 years back: https://github.com/bitcoin-dot-org/developer.bitcoin.org. Only the code in this repo should be treated as source of truth.

Reply:
These docs are indeed outdated, last update was 4 years back: https://github.com/bitcoin-dot-org/developer.bitcoin.org. Only the code in this repo should be treated as source of truth.